dream_layer_backend/comfy_nodes/api_nodes/runway_text2img.py

```
@@ -0,0 +1,326 @@
+
+API Reference: https://docs.dev.runwayml.com/api/#tag/Start-generating/paths/~1v1~1text_to_image/post
+"""
+
+import os
+import httpx
+import torch
+import numpy as np
+import time
+from PIL import Image
+from io import BytesIO
+from typing import Optional, List, Dict, Any
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import

(F401)